src/main/resources/kibana-rest.yml

```
@@ -20,7 +24,7 @@ kibana_actions:
 
 kibana_alerts:
   versions:
-    ">= 7.9.0": "/api/alerts/_find"
+    ">= 7.9.0": "/api/alerts/_find#spaceaware#paginate[per_page]"
```

Or perhaps to future-proof it:

```suggestion
kibana_alerts:
  versions:
    ">= 7.9.0": "/api/alerts/_find#spaceaware#paginate[per_page]"
      paginate: per_page
      spaceaware: true
```

---

I can go for this one

---

Actually, this is not valid YAML - I have to go with:
```
kibana_alerts:
  versions:
    ">= 7.9.0":
      api: "/api/alerts/_find#spaceaware#paginate[per_page]"
      paginate: per_page
      spaceaware: true
```

---

I've submitted a change which allows both the old format and the new format.

---

Thanks. I'll review this today.